docs/src/docs/asciidoc/gradle-plugin-quickstart.adoc

```diff
@@ -6,39 +6,39 @@ This guide shows how to get started with the <<gradle-plugin.adoc#,Gradle plugin
 
 You will create a sample application, enable the plugin, add support for dynamic features, run JUnit tests, and build a native executable.
 
-Two ways to build a native executable using the plugin are demonstrated:
+Two ways of building a native executable using the plugin will be demonstrated:
 
 - <<#build-a-native-executable-with-resources-autodetection,Build a Native Executable with Resources Autodetection>>
 -  <<#build-a-native-executable-detecting-resources-with-the-agent,Build a Native Executable Detecting Resources with the Agent>>
 
-[NOTE]
 ====
-.Sample Application
-
-You start by creating a **Fortune Teller** sample application that simulates the traditional 
-https://en.wikipedia.org/wiki/Fortune_(Unix)[fortune Unix program]. 
-The data for the fortune phrases is provided by https://github.com/your-fortune[YourFortune].
+The plugin requires that you <<graalvm-setup.adoc#,setup GraalVM>>.
+
+The easiest way to install GraalVM is to use the https://sdkman.io/jdks[SDKMAN!].
+To install GraalVM for JDK 20 or JDK 17, run:
+```
+sdk install java 20.0.1-graal
+sdk install java 17.0.7-graal
+```
+GraalVM installation includes Native Image.
+For other installation options, go to https://www.graalvm.org/downloads/[GraalVM Downloads].
 ====
 
-====
-To use the Native Build Tools, install GraalVM with Native Image. 
-The easiest way to install GraalVM with Native Image is to use the https://github.com/graalvm/graalvm-jdk-downloader[GraalVM JDK Downloader]:
+== Prepare a Demo Application
 
-[source,bash]
-----
-bash <(curl -sL https://get.graalvm.org/jdk)
-----
 ====
-
-== Prepare a Demo Application
+You start by creating a **Fortune Teller** sample application that simulates the traditional
+https://en.wikipedia.org/wiki/Fortune_(Unix)[fortune Unix program].
+The data for the fortune phrases is provided by https://github.com/your-fortune[YourFortune].
+====
 
 . Create a new Java project with *Gradle* using the following command (alternatively, you can use your IDE to generate a project):
 +
 [source,shell]
 ----
 gradle init --project-name fortune-parent --type java-application --package demo --test-framework junit-jupiter --dsl groovy
 ----
-. Rename the default `app` directory to `fortune`, edit the `settings.gradle` file to replace `app` with `fortune`, then rename the default filename `App.java` to `Fortune.java`, and replace its contents with the following:
+. Rename the default _app_ directory to _fortune_, edit the _settings.gradle_ file to replace `app` with `fortune`, then rename the default filename _App.java_ to _Fortune.java_, and replace its contents with the following:
```

Comment:
Curious why you replaced the backticks with underscores? This will change the formatting from monospace to regular. I think the use of backticks is correct in the context.

Reply:
We have a convention with @bhoran  to use italicized font for the directories and paths, unlike monospace for commands and code. See the preview: 
![Screenshot 2023-07-03 at 13 59 34](https://github.com/graalvm/native-build-tools/assets/31922163/1e95c29c-852b-4c7c-b76b-fa12ea948e8f)